src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -51,40 +46,42 @@ import {
 import { Sticky } from "v2/Components/Sticky"
 import { ScrollRefContext } from "./ArtworkFilters/useScrollContext"
 import { ArtworkSortFilter } from "./ArtworkFilters/ArtworkSortFilter"
-import { GeneArtworkFilter_gene } from "v2/__generated__/GeneArtworkFilter_gene.graphql"
 import type RelayModernEnvironment from "relay-runtime/lib/store/RelayModernEnvironment"
-import { TagArtworkFilter_tag } from "v2/__generated__/TagArtworkFilter_tag.graphql"
-import { Works_partner } from "v2/__generated__/Works_partner.graphql"
-import { CollectionArtworksFilter_collection } from "v2/__generated__/CollectionArtworksFilter_collection.graphql"
 import { ArtworkGridFilterPills } from "./SavedSearch/Components/ArtworkGridFilterPills"
 import { SavedSearchAttributes } from "./SavedSearch/types"
 import { extractPills } from "../SavedSearchAlert/Utils/extractPills"
 import { useFilterPillsContext } from "./SavedSearch/Utils/FilterPillsContext"
 import { getTotalSelectedFiltersCount } from "./Utils/getTotalSelectedFiltersCount"
 
+interface ArtworkFilterProps extends SharedArtworkFilterContextProps, BoxProps {
+  enableCreateAlert?: boolean
+  Filters?: JSX.Element
+  offset?: number
+  // Input variables passed to FilterArtworkConnection `input` argument
+  relayRefetchInputVariables?: object
```

Comment:
Realized we had no way to control how refetches work after the initial filter render, when we call `prepareVariables` from the routers file, so added that here and cleaned up some types below.